host/sgx/windows/debugrtbridge.c

```diff
@@ -47,18 +47,19 @@ static void load_oedebugrt(void)
             debugrtpath,
             NULL, /* reserved */
             /* Search only specified path. */
-            LOAD_LIBRARY_SEARCH_DEFAULT_DIRS);
+            LOAD_WITH_ALTERED_SEARCH_PATH);
```

Comment:
`LOAD_WITH_ALTERED_SEARCH_PATH` still has the fallback to system directories right? If that's the intention, then the comment should be updated.

Reply:
From https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryexa

> If the string specifies a **fully qualified path, the function searches only that path for the module**. When specifying a path, be sure to use backslashes (), not forward slashes (/). For more information about paths, see Naming Files, Paths, and Namespaces.

My interpretation of the doc is that if a full path is specified, only that exact path is searched for the module. In such a scenario, the flags LOAD_WITH_ALTERED_SEARCH_PATH, LOAD_LIBRARY_SEARCH_DEFAULT_DIRS affect the search order for the modules that the loaded dll depends on.

In any case I am going to remove this debug section.